tests/ct.yaml

```diff
@@ -0,0 +1,4 @@
+remote: upstream
+target-branch: master
+chart-dirs:
+  - charts
```

Comment:
What's the purpose of this file?

Reply:
this is the ct lint config file https://github.com/helm/chart-testing#config-file